examples/main.go

```diff
@@ -71,14 +72,14 @@ func main() {
 
 	switch r := resp.(type) {
 	case *momento.CacheGetHit:
-		log.Printf("Lookup resulted in cahce HIT. value=%s\n", r.ValueString())
+		mLogger.Info("Lookup resulted in cahce HIT", []string{"key", key, "value", r.ValueString()}...)
```

Comment:
Typo in `cahce` also shouldn't we be able to just pass additional args to log like this now?
```suggestion
		mLogger.Info(
		   "Lookup resulted in cache HIT", "key", key, "value", r.ValueString(),
	         )
```

Reply:
Thank you for catching it!
Dah! I will update them.